app/services/widgets/settings/alert-box/alert-box-data.ts

```diff
@@ -23,22 +23,29 @@ export const API_NAME_MAP = {
   fanfunding: 'fanfunding',
   subscriber: 'subscribers', // YouTube
   sponsor: 'sponsors',
+  facebook_support_gifter: 'giftSupport',
   facebook_support: 'support',
   facebook_like: 'likes',
   facebook_stars: 'stars',
   facebook_share: 'shares',
   facebook_follow: 'fbfollows',
   loyalty_store_redemption: 'loyaltystore',
-  effect: 'effects',
-  sticker: 'stickers',
+};
+
+const determineTester = (key: string) => {
```

Comment:
Can you add a comment about this method? I'm still not 100% sure why we have different testers here

Reply:
can do